spec/features/user_register_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+
+describe "the signup process", :type => :feature, :js => true do
+
+  context 'with proper params' do
+
+    it "signs me up" do
+      visit '/user_registrations/new'
+      fill_in 'email', :with => 'user@example.com'
+      fill_in 'password', :with => 'password'
+      fill_in 'password_confirmation', :with => 'password'
+
+      click_button 'Sign up'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.